Beam/Include/Beam/Queues/FilterWriterQueue.hpp

```diff
@@ -81,13 +81,13 @@ namespace Beam {
     if(!m_filter(value)) {
       return;
     }
-    m_target->Push(value);
```

Comment:
This is embarrassing, I will need to boost test coverage to avoid these issues in the future.